requirements/requirements_dbt_0.18.x.txt

```diff
@@ -7,4 +7,5 @@ MarkupSafe==2.0.1
 PyYAML>=5.1
 pyopenssl<20.0.0
 pytz<2021.0
+ruamel.yaml>=0.17.12
```

Comment:
Just curious if version `0.17.12` has significance or if it's just what you already had locally.

Reply:
jup the latter 🙃

Reply:
but 0.17.12 is also the current version https://pypi.org/project/ruamel.yaml/